### PR TITLE
Todo add - remove function

### PR DESCRIPTION
**Pull request by yusufobr**
In this Pull Request : 
- Implement adding & removing functions.
- Implement editing task.
- Implement deleting tasks that are done.
- Implement local storage.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -6,18 +6,6 @@ env:
...
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup Lighthouse
-        run: npm install -g @lhci/cli@0.7.x
-      - name: Lighthouse Report
-        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
```
Comment (Shedrack-Sunday) [inline .github/workflows/linters.yml]: - [ ] You're not meant to alter any sections of your linters file. Kindly return the deleted section you removed 👍

File: README.md [hunk 2]
```diff
@@ -51,6 +51,7 @@
 ### Key Features <a name="key-features"></a>
 
 - **[Use JavaScript to add and remove to-do items]**
+- **[Use Javascript to edit tasks]**
 - **[Use webpack to manage the code]**
 - **[Use css to modify the appearance of the list]**
 - **[Use ES6 to modify the javaScript code]**
```
Comment (Shedrack-Sunday) [inline README.md:57]: - [ ] I tried to access your MIT license, and it says "not added on `add-remove`". Kindly fix this to make it accessible 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @yusufobr        👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights
✔️ Good PR summary
✔️ Nice use of Javascript for editing, adding, and deleting
✔️ Good use of Webpack

## Confirmed functionality:
image

  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below need to be implemented :
- Kindly fix your MIT license
- You deleted some linter sections. Kindly put it back

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 2 above)
Comment (yusufobr) [reply README.md:57]: done

**Review by mwanawabangona — APPROVED** [review]
Hi @yusufobr,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*  Good job implementing the rquested changes


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**